lib/github/kv.rb

```diff
@@ -299,20 +303,12 @@ def validate_key_array(keys)
 
     def validate_key_value_hash(kvs)
       unless kvs.is_a?(Hash)
-        raise TypeError, "kvs must be a {String => String} in #{self.class.name}, but was #{key.class}"
+        raise TypeError, "kvs must be a {String => String} in #{self.class.name}, but was #{kvs.class}"
       end
 
       kvs.each do |key, value|
-        unless key.is_a?(String)
-          raise TypeError, "kvs must be a {String => String} in #{self.class.name}, but also saw at least one key of type #{key.class}"
-        end
-
-        unless value.is_a?(String)
-          raise TypeError, "kvs must be a {String => String} in #{self.class.name}, but also saw at least one value of type #{value.class}"
-        end
-
-        validate_key_length(key)
-        validate_value_length(value)
+        validate_key(key, error_message: "kvs must be a {String => [String|Literal]} in #{self.class.name}, but also saw at least one key of type #{key.class}")
```

Comment:
`[String|Literal]` => Should this be `[String|SQL::Literal]`?

Reply:
It should! Good catch! :bow: